mods/ra/maps/exodus/map.yaml

```diff
@@ -0,0 +1,1429 @@
+	Actor189: ftur
+		Location: 147,64
+		Owner: Soviets
+	Actor191: barl
+		Location: 160,55
+		Owner: Soviets
+	SovietAirfield1: afld
+		Location: 151,69
+		Owner: Soviets
+	SovietAirfield2: afld
+		Location: 154,69
+		Owner: Soviets
+	Actor228: e3
+		Location: 158,66
+		Owner: Soviets
+	Actor241: barl
+		Location: 157,67
+		Owner: Soviets
+	Actor242: barr
+		Location: 159,61
+		Owner: Soviets
+	Actor56: tc01
+		Location: 145,71
+		Owner: Neutral
+	Actor237: dog
+		Location: 151,62
+		Owner: Soviets
+	Actor11: e1
+		Location: 149,64
+		Owner: Soviets
+	Actor192: fenc
+		Location: 147,65
+		Owner: Soviets
+	Actor244: e1
+		Location: 160,60
+		Owner: Soviets
+	Actor206: fenc
+		Location: 158,53
+		Owner: Soviets
+	Actor18: apwr
+		Location: 149,72
+		Owner: Soviets
+	Actor230: 3tnk
+		Location: 157,60
+		Owner: Soviets
+	Actor231: kenn
+		Location: 162,59
+		Owner: Soviets
+	Actor235: waypoint
+		Location: 147,51
+		Owner: Neutral
+	Actor205: fenc
+		Location: 159,53
+		Owner: Soviets
+	Actor207: fenc
+		Location: 158,52
+		Owner: Soviets
+	Actor208: fenc
+		Location: 157,52
+		Owner: Soviets
+	Actor204: fenc
+		Location: 147,70
+		Owner: Soviets
+	Actor193: fenc
+		Location: 147,66
+		Owner: Soviets
+	Actor195: fenc
+		Location: 147,67
+		Owner: Soviets
+	Actor197: fenc
+		Location: 146,67
+		Owner: Soviets
+	Actor199: fenc
+		Location: 146,69
+		Owner: Soviets
+	Actor198: fenc
+		Location: 146,68
+		Owner: Soviets
+	Actor202: fenc
+		Location: 147,69
+		Owner: Soviets
+	Actor292: e2
+		Location: 165,59
+		Owner: Soviets
+	Actor232: t01
+		Location: 72,43
+		Owner: Neutral
+	Actor176: tc05
+		Location: 69,46
+		Owner: Neutral
+	Actor170: mine
+		Location: 114,55
+		Owner: Neutral
+	Actor169: mine
+		Location: 113,57
+		Owner: Neutral
+	Allies2MovePoint: waypoint
+		Location: 167,38
+		Owner: Neutral
+	Actor3: silo
+		Location: 172,59
+		Owner: Soviets
+	Actor33: fact
+		Location: 162,67
+		Owner: Soviets
+	SovietEntryPoint6: waypoint
+		Location: 53,79
+		Owner: Neutral
+	SovietRallyPoint6: waypoint
+		Location: 53,75
+		Owner: Neutral
+	Actor171: mine
+		Location: 168,73
+		Owner: Neutral
+	Actor248: ftur
+		Location: 64,44
+		Owner: Soviets
+	SovietAirfield7: afld
+		Location: 62,49
+		Owner: Soviets
+	Actor256: ftur
+		Location: 74,57
+		Owner: Soviets
+	Actor273: brik
+		Location: 72,61
+		Owner: Soviets
+	Actor44: t11
+		Location: 57,54
+		Owner: Neutral
+	Actor249: t11
+		Location: 98,58
+		Owner: Neutral
+	Actor245: tc05
+		Location: 106,43
+		Owner: Neutral
+	Actor233: ftur
+		Location: 75,54
+		Owner: Soviets
+	Actor287: brik
+		Location: 72,60
+		Owner: Soviets
+	Actor258: brik
+		Location: 71,61
+		Owner: Soviets
+	Actor293: brik
+		Location: 70,61
+		Owner: Soviets
+	Actor177: tc04
+		Location: 59,50
+		Owner: Neutral
+	Actor294: brik
+		Location: 72,59
+		Owner: Soviets
+	Actor213: tsla
+		Location: 71,59
+		Owner: Soviets
+	Actor295: brik
+		Location: 73,59
+		Owner: Soviets
+	Actor180: v01
+		Location: 100,29
+		Owner: Soviets
```

Comment:
I assume it is a mistake in the original map design, a rather ironic one as well. Fixed.